h2o-r/h2o-package/pkgdown/_pkgdown.yml

```diff
@@ -244,6 +245,7 @@ reference:
       - h2o.permutation_importance
       - h2o.permutation_importance_plot
       - h2o.pivot
+      - h2o.plot_gains_lift
```

Comment:
I don't like the fact that we changed the naming convention here.
Until now:
```
     - h2o.permutation_importance
     - h2o.permutation_importance_plot
```
(same in Py: `model.varimp` vs. `model.varimp_plot`, `model.std_coef` vs. `model.std_coef_plot` …)
With this PR:
```
- h2o.gains_lift
- h2o.plot_gains_lift
```

We need to be consistent. And actually I prefer the legacy style (easily discovered during autocomplete when first trying to get the `gains_lift`)

Reply:
Makes sense, I made the appropriate changes (`plot_gains_lift -> gains_lift_plot`).

Reply:
Agreed @sebhrusen, thanks for the name adjustment @tomasfryda.